aws/table_aws_ses_email_identity.go

```diff
@@ -0,0 +1,195 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableAwsSESEmailIdentity(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ses_email_identity",
+		Description: "AWS SES Email Identity",
+		List: &plugin.ListConfig{
+			Hydrate: listSESEmailIdentities,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The email identity.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "verification_status",
+				Description: "The verification status of the identity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEmailIdentityVerificationAttributes,
+			},
+			{
+				Name:        "verification_token",
+				Description: "The verification token for a domain identity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEmailIdentityVerificationAttributes,
+			},
+			{
+				Name:        "notification_attributes",
+				Description: "Represents the notification attributes of an identity.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEmailIdentityNotificationAttributes,
+				Transform:   transform.FromValue(),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "arn",
```

Comment:
`arn` should be the 2nd column. This column is not part of Steampipe standard columns.